source/_integrations/flux_led.markdown

```diff
@@ -10,7 +10,9 @@ ha_platforms:
   - light
 ---
 
-The `flux_led` support is integrated into Home Assistant as a light platform. Several brands of both bulbs and controllers use the same protocol and they have the HF-LPB100 chipset in common. The chances are high that your bulb or controller (eg. WiFi LED CONTROLLER) will work if you can control the device with the MagicHome app.
+The `Flux_LED` support is integrated into Home Assistant as a light platform. Several brands of both bulbs and controllers use the same protocol and they have the HF-LPB100 chipset in common. The chances are high that your bulb or controller (eg. WiFi LED CONTROLLER) will work if you can control the device with the MagicHome app.
```

Comment:
We can remove the `Flux_LED` and just use normal words now, as this is no longer a platform. Also, remove the platform reference in the first sentence for the same reason.